src/reducers/auth.js

```diff
@@ -2,7 +2,8 @@ import * as types from '../actions/types';
 
 const INITIAL_STATE = {
   authenticated: false,
-  user: {},
+  user: null,
+  token: null,
```

Comment:
`user` should have user.token now because of `LOGIN_GUCCI`

Remove `token` from the initial state since it's not being used. We'll use `user.token` instead for clarity